meiki_ui/src/components/Button.svelte

```diff
@@ -1,15 +1,18 @@
 <script context="module" lang="ts">
     export enum ButtonType {
+        HERO,
         PRIMARY,
```

Comment:
wonder if we should call something other than Primary/Secondary (especially secondary)

Reply:
Hmm, yep we just kind of assumed this, we don't know what it really means. Will look into UI jargons for this.